Assets/Scripts/Cristopher's/ChooseFactionScript1.cs

```diff
@@ -0,0 +1,72 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
```

Comment:
Please change naming to sth more understandable than "<scriptName>1" and "<scriptName>2"